### ISSUE TITLE
Update Symfony to 4.2 and Flex

### ISSUE THREAD

**Issue by mnapoli** [mnapoli/bref-bootstrap-benchmarks#1]

**Comment by bubba-h57** [comment]
I am using Symfony 4.2 and Flex ^1.1 in #8 ... what is it that Flex needs to be updated to?

**Comment by mnapoli** [comment]
Unless I'm mistaken all Symfony examples in this repository are using 4.1.

**Comment by bubba-h57** [comment]
Yes, but my pull request, I used 4.2

**Comment by mnapoli** [comment]
Right, I think it's better if we change one thing at a time, and if we keep all benchmarks similar (same Symfony version). Could you change it to use the same `composer.json` and Symfony config as others?

Then we can update all benchmarks at once.
